tests/Unit/Schema/Types/LaravelEnumTypeTest.php

```diff
@@ -34,6 +34,13 @@ public function testMakeEnumWithCustomName(): void
         $this->assertSame($customName, $enumType->name);
     }
 
+    public function testEnumDescription(): void
+    {
+        $enumType = new LaravelEnumType(LocalizedUserType::class);
+
+        $this->assertSame('Localized user type', $enumType->config['description']);
```

Comment:
Will not work in every case in the test matrix, also needs to check if `getClassDescription` exists and assert accordingly

Reply:
Thanks, I was wondering how to deal with it in this case 👍